theme/sass/_variables.scss

```diff
@@ -1,6 +1,13 @@
+// Colors.
 $primary-color: #e67e70;
 $secondary-color: #363636;
 $tertiary-color: #7BBABA;
 $accent-color-one: #f5f5f5;
-$accent-color-two: #363636;
-$gutter-size: 90%;
\ No newline at end of file
+
+// Breakpoint Variables.
+$sm: 576px;
+$md: 768px;
+//$lg: 992px;
```

Comment:
This $lg breakpoint will need uncommenting if you're going to be using it for the early mobile header change.

Reply:
already done